### PR TITLE
Add Profile picture

### PR DESCRIPTION
**Pull request by kschmitz123**
toDo:

- [x] remove image upload button when logged in user is not owner of the profile
- [x] solve mongodb issue: suddenly a new user is created although it already exists

### CODE REVIEW

File: client/src/components/Profile.js [hunk 1]
```diff
@@ -1,29 +1,147 @@
 import styled from "styled-components/macro";
```
Comment (boldCM) [inline client/src/components/Profile.js:1]: Maybe I'm  mistaken but shouldn't you also `import React from 'react' ` ?

**Review by boldCM — APPROVED** [review]
lgtm 🎉

File: lib/models/users.js [hunk 2]
```diff
@@ -13,6 +14,9 @@ async function setUser(data) {
 async function getUser({ username }) {
   return await collection("users").findOne({ username: username });
 }
+async function getUserImage(username) {
+  return await collection("users").findOne({ username: username });
+}
```
Comment (mhstrkmp) [inline lib/models/users.js:19]: Looks more or less the same than the getUser function. Maybe you can just use one function for both.

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

File: client/src/components/Profile.js (hunk 1 above)
Comment (kschmitz123) [reply client/src/components/Profile.js:1]: don't need it anymore :)

File: lib/models/users.js (hunk 2 above)
Comment (kschmitz123) [reply lib/models/users.js:19]: i tried but then i got an error, because in one i deconstruct the user and in the other not. Happy for suggestions :)

Comment (mhstrkmp) [reply lib/models/users.js:19]: Not sure but maybe you could destructure the user in the [function call](https://github.com/kschmitz123/diy-project-app/blob/52a2a0670b6b82d7ca89a105a2f291cdb8d5dcf8/lib/routes/users.js#L10)

Maybe something like this: `const {user} = await getUser(user);`

**Pull request merged**